spec/commands/v2/put_content_spec.rb

```diff
@@ -117,6 +117,29 @@
         expect(State.find_by!(content_item: content_item).name).to eq("draft")
         expect(UserFacingVersion.find_by!(content_item: content_item).number).to eq(6)
       end
+
+      describe "race condtitions", skip_cleaning: true do
+        after do
+          DatabaseCleaner.clean_with :truncation
+        end
+
+        it "copes with race conditions" do
+          described_class.call(payload)
+          Commands::V2::Publish.call(content_id: content_id, update_type: "minor")
+
+          begin
```

Comment:
This could be:
```ruby
expect {
  thread1 = Thread.new { described_class.call(payload) }
  thread2 = Thread.new { described_class.call(payload) } 
  thread1.join
  thread2.join
}.to raise_error ExceptionClass
```